services/movie_session.py

```diff
@@ -41,4 +42,17 @@
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    try:
+        movie_session = MovieSession.objects.get(id=session_id)
+        movie_session.delete()
+    except ObjectDoesNotExist:
+        raise ValueError(f"MovieSession with id {session_id} does not exist.")
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
```

Comment:
The function `get_taken_seats` correctly handles the case where the movie session does not exist by raising a `ValueError`. This is a good practice to inform the caller about the non-existence of the session.